CONTRIBUTING.md

```diff
@@ -37,9 +37,12 @@ Code changes to Gleam are welcomed via the process below.
    the new functionality should work.
 2. Open a GitHub pull request with your changes and ensure the tests and build
    pass on CI.
-3. A Gleam team member will review the changes and may provide feedback to
+3. Reflect your changes in [CHANGELOG.md](CHANGELOG.md) under Unreleased at the top of the file.
```

Comment:
You may want to do this last, or to recommend to put it in the middle, to avoid merge conflicts